Assets/AnkrSDK/Scripts/Data/Wallets.cs

```diff
@@ -0,0 +1,10 @@
+namespace AnkrSDK.Data
+{
+	public enum Wallet
```

Comment:
file name is different

Reply:
also why did you delete  
        Argent,
        Rainbow,
        Pillar
        from the list?

Reply:
I didn't delete anything. It was a file with WebGL supported wallets.

Reply:
This file name is Wallets.cs but enum is named Wallet.
also there is another file AnkrSDK.WalletConnectSharp.Unity.Models.DeepLink.Wallets
please make sure there is no confusion between these.

Reply:
I got about file name.